### ISSUE TITLE
After selecting zh-cn, the SCM display language is (en) but not (zh-cn)

### ISSUE THREAD

**Issue by missuor** [microsoft/vscode-loc#108]
```
版本: 1.38.1
提交: b37e54c98e1a74ba89e03073e5a3761284e3ffb0
日期: 2019-09-11T13:30:08.229Z
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Linux x64 5.0.0-27-generic
```
Before:
extensions folder not exists in /xxx/.vscode/extensions/ms-ceintl.vscode-language-pack-zh-hans-1.38.3/translations/
![image](https://user-images.githubusercontent.com/4561294/64746418-04feb200-d53e-11e9-808a-c593a7c1cb1b.png)

ln -ns $HOME/.vscode/extensions/ms-ceintl.vscode-language-pack-zh-hans-1.38.3/translations/.build/extensions $HOME/.vscode/extensions/ms-ceintl.vscode-language-pack-zh-hans-1.38.3/translations/extensions
Restart vscode
After:
![image](https://user-images.githubusercontent.com/4561294/64746600-d9c89280-d53e-11e9-9a0e-3e172ad49433.png)

**Comment by nevertback** [comment]
我也遇到这个问题了，把/extensions文件夹从/.build拷出来就好了
![image](https://user-images.githubusercontent.com/4335773/64747481-bf43e880-d541-11e9-810e-979865bc9225.png)

**Comment by ShawDanon** [comment]
mac版也是如此

**Comment by janbaltus** [comment]
Fixed in 1.38.4 LP.
